js/views/userPageVw.js

```diff
@@ -322,15 +318,29 @@ module.exports = baseVw.extend({
     });
   },
 
-  bindModalCloseHandler: function(e) {
-    if (this.modalCloseHandler) return;
-
-    this.modalCloseHandler = function() {
-      window.history.back();
+  loadingStatus: function() {
+    var config = {
+      promise: this.loadingDeferred.promise(),
+      cancel: () => {
+        this.userProfileFetch && this.userProfileFetch.abort();
+        this.itemFetch && this.itemFetch.abort();
+      }
     };
+    
+    if (this.currentItemHash) {
+      config.connectText = window.polyglot.t('pageConnectingMessages.listingConnect').replace('${listing}', this.currentItemHash);
```

Comment:
The messaging properties you can return (optionally) are:
`connectText` - text when the page is connecting
`failedText` - text when the page failed to connect
`connectTooltip` - tooltip when the page is connecting
`failedTooltip`  - tooltip when the page failed to connect